### DISCUSSION TITLE
Not able to understand the context of a feature in the site

### DISCUSSION THREAD

**Discussion by aditya123473892** [processing/p5.js-web-editor#2790] [General]
![error](https://github.com/processing/p5.js-web-editor/assets/117269123/6544d9cc-34d8-4b34-b7a3-47add8d31362)
the content present in the red block does not have any context why is it used for. Writing a  text to explain what the feature is about this will help the users to use it more often and understand it better .

### Feature enhancement details

Adding a text just next to the feature might help it or adding new design for it  . I don't have a exact idea how to fix it  suggestions might help it in resolving  it

**Comment by SasiDharan731** [comment]
I believe that adding a simple tooltip would be helpful, like this

<img width="1359" alt="Screenshot 2023-12-25 at 4 47 02 PM" src="https://github.com/processing/p5.js-web-editor/assets/56504102/14bab761-3b22-4dce-bbb1-df106432b949">

**Reply by lindapaiste** [reply]
I think that a tooltip is a good compromise.  I'm generally against having a lot of labels because in my opinion it clutters the UI.

This element is interactive so maybe "Edit sketch name" or "Rename your sketch" makes more sense than just "Sketch name"?

**Comment by aditya123473892** [comment]
we should ask the maintainers about it how to  fix or  does it needs to be fixed or not in their opinion. @lindapaiste sir does this issue should be worked on or not

**Comment by ayushanand308** [comment]
@aditya123473892  It generates random names using the "friendly-words" npm package. These names are designed to be memorable and easy to spell, making them suitable for sketch names. You can find more information on how the package works by visiting its GitHub repository: [https://github.com/glitchdotcom/friendly-words.](url)
As suggested by @SasiDharan731 it might be useful to add a tooltip.

**Reply by mkmori** [reply]
I've expressed my opinion about the random names feature at length elsewhere: As fun & whimsical as it might be at times, this issue seems to me an example of the sort of mild confusion it tends to cause.

For artistic, cultural, linguistic, and utilitarian reasons, I would prefer by default a boring, neutral, but self-explanatory naming scheme, like that used in PDE: sketch_YYMMDDa

The random words feature might be optionally, intentionally invoked by users specifically looking for some random inspiration. And if it suggested individual terms (rather than modifier-object pairs), it would probably not have to rely on the rather impoverished (IMO) "friendly-words" lists, which only seem bound to contract over time....

For what it's worth! :)
